src/syntax/grammar.mly

```diff
@@ -1668,6 +1673,9 @@ and parse_call_params f p1 s =
 			| [< '(PClose,p2) >] ->
 				let e = check_signature_mark e p1 p2 in
 				f (List.rev (e :: acc)) p2
+			| [< '(Comma,p2); '(PClose,p3) >] ->
+				let e = check_signature_mark e p1 p3 in
+				f (List.rev (e :: acc)) p3
```

Comment:
```ocaml
| [< '(PClose,p2) >]
| [< '(Comma,_); '(PClose,p2) >] ->
	let e = check_signature_mark e p1 p2 in
	f (List.rev (e :: acc)) p2
```

Maybe? (that works in ocaml, right?)

Reply:
Doesn't seem to work like that :/

Reply:
Yea, even `'(Comma,_);` has different result for match. Maybe even this thing has special behavior in caml5p extension